src/pages/workspace/content/review/index.tsx

```diff
@@ -171,6 +188,10 @@ const ReviewContentSubmissions = () => {
   };
   return (
     <Card sx={{ padding: 2, backgroundColor: "white" }}>
+           { publishOpenToast && (<ToastNotification message="Published Content Successfully" type= "success" />)}
```

Comment:
@AkshataKatwal16 change message to `Content published successfully`

Reply:
done @itsvick